wiki/Help_centre/Account_restrictions/vi.md

```diff
@@ -74,48 +68,58 @@ Nếu chúng tôi quyết định cho bạn quay lại, chúng tôi sẽ yêu c
+- **Khiếu nại của bạn sơ sài hoặc chưa hoàn chỉnh.** Nếu khiếu nại của bạn không đề cập đủ bốn điểm ở trên, chúng tôi sẽ yêu cầu bạn viết lại khiếu nại khác đầy đủ hơn. Việc này **không** khiến thời gian chờ khiếu nại của bạn bị đặt lại và bạn có thể thoải mái gửi khiếu nại mới cho chúng tôi vào bất cứ lúc nào! Tuy nhiên, nếu bạn có ý định dối trá hay giấu đi những lần bạn phạm quy nghiêm trọng (thường là chơi nhiều tài khoản hoặc gian lận), khiếu nại của bạn sẽ bị từ chối vì thiếu trung thực, và thời gian chờ sẽ bị đặt lại về ít nhất **ba tháng**. Đây là cơ hội khác cho bạn hối cải về hành vi của mình và đảm bảo bạn không quên gì cả.
+- **Bạn đang né tránh hình phạt hoặc chơi trên tài khoản khác.** Tạo tài khoản khác để chơi trực tuyến khi tài khoản của bạn bị hạn chế được coi là một hành động né tránh hạn chế và nó sẽ hoàn lại thời gian chờ về ít nhất **ba tháng** kể từ ngày bạn tạo tài khoản mới. Nếu bạn vẫn cố trốn tránh, khiếu nại của bạn có thể bị từ chối vĩnh viễn. Nếu bạn vẫn truy cập và chơi trên tài khoản phụ khi bị hạn chế, thời gian chờ cũng sẽ bị hoàn lại. Bạn hãy nhớ là dùng nhiều tài khoản là vi phạm [nội quy](/wiki/Rules) và chúng tôi hi vọng bạn chỉ chơi trên tài khoản gốc. Nếu bạn không rõ tài khoản nào là tài khoản gốc của bạn, hãy gửi email và chúng tôi sẽ cho bạn biết!
 - **Bạn tiếp tục vi phạm khi đang bị hạn chế.** Không có lý do gì để gỡ hạn chế một người vẫn cố vi phạm kể cả khi đã bị hạn chế. Nếu bạn tiếp tục tái phạm, thời gian chờ sẽ bị hoàn lại kể từ lần phạm quy gần nhất. Trong trường hợp vi phạm nghiêm trọng hoặc tía diễn liên tục, khiếu nại của bạn có thể bị từ chối vĩnh viễn.
 - **Lịch sử vi phạm quá nghiêm trọng.** Trong trường hợp phạm lỗi nghiệm trọng, hoặc người chơi bị hạn chế lần nữa sau khi đã được gỡ hạn chế bởi một lần khiếu nại trước đó, chúng tôi có thể cho rằng trường hợp của bạn là quá nghiêm trọng để có thể quay trở lại game vào thời điểm hiện tại. Nếu bạn thuộc diện này, chúng tôi khuyến nghị bạn nên chờ **một thời gian dài** trước khi khiếu nại, và đảm bảo rằng bạn không vi phạm thêm trong thời gian đó.
 
+## Điều gì xảy ra nếu đây không phải lần đầu tôi khiếu nại?
+
+Với hầu hết trường hợp, mỗi lần tài khoản bạn bị hạn chế sẽ gấp đôi thời gian bạn bị hạn chế lần tiếp theo và cả thời gian chờ khiếu nại.
+
+Ví dụ, bạn bị hạn chế vì gian lận, quay lại sau khiếu nại, sau đó gian lận tiếp, bạn sẽ bị hạn chế tối thiểu 12 tháng (gấp đôi khoảng sáu tháng gốc) trước khi chúng tôi xem xét bất kỳ khiếu nại nào từ bạn. Nếu bạn quay lại và lại gian lận tiếp, bạn sẽ bị hạn chế tối thiểu 24 tháng.
+
+Ghi chú thêm, việc quay lại sau lần vi phạm đầu tiên sẽ trở nên vô cùng khó khăn, vì chúng tôi phải cân nhắc những hành vi trong quá khứ của bạn và việc gỡ hạn chế cho bạn có đáng hay không. Rất ít (nếu có) người chơi trở về thành công sau lần hạn chế thứ ba.
```

Comment:
```suggestion
Ghi chú thêm, việc quay lại sau lần vi phạm đầu tiên sẽ trở nên vô cùng khó khăn, vì chúng tôi phải cân nhắc những hành vi trong quá khứ của bạn và việc gỡ hạn chế cho bạn có đáng về thời gian và rủi ro đem lại hay không. Rất ít (nếu có) người chơi trở về thành công sau lần hạn chế thứ ba.
```

Reply:
I personally think shorten this part is okay.
"is worth or not" mentions a broad case, rather than just `"time" and "risk" involved`.